newIDE/app/src/AssetStore/ExtensionStore/ExtensionStoreContext.js

```diff
@@ -39,6 +42,9 @@ export const ExtensionStoreContext = React.createContext<ExtensionStoreState>({
   error: null,
   searchText: '',
   setSearchText: () => {},
+  allCategories: [],
```

Comment:
Should it be null so that it's clear categories were not fetched yet

Reply:
It would force to add a check when building the drop-down list.
If we don't display the drop-down list while the registry is retried, won't that makes the UI jumpy?

Reply:
We could still have it null and display a SelectField that has a single, disabled option called "Loading..." (I agree with you that we don't want a jumpy UI).

BUT as this is maybe more work for not a lot (and we can do this later), I'm ok to avoid this (other components are taking care of showing a proper loader).